docs/guides/compile-contracts.md

```diff
@@ -132,7 +134,7 @@ You shouldn't interact with these files directly.
 
 The [HRE] has an `artifacts` object with helper methods. For example, you can get a list with the paths to all artifacts by calling `hre.artifacts.getArtifactPaths()`.
 
-You can also read an artifact using the name of the contract by calling `hre.artifacts.readArtifact("Bar")` and that will give us the content of the artifact for the `Bar` contract. This would only work if there was just one contract `Bar` in the whole project, but calling `hre.artifacts.readArtifact("Foo")`, would throw an error if there were two `Foo` contracts. To disambiguate this case, you would have to use the **Fully Qualified Name** of the contract: `hre.artifacts.readArtifact("contracts/Foo.sol:Foo")`.
+You can also read an artifact using the name of the contract by calling `hre.artifacts.readArtifact("Bar")` and that will yield the content of the artifact for the `Bar` contract. This would only work if there was just one contract named `Bar` in the whole project; it would throw an error if there were two. To disambiguate this case, you would have to use the **Fully Qualified Name** of the contract: `hre.artifacts.readArtifact("contracts/Bar.sol:Bar")`.
```

Comment:
I think I'd use return instead of yield, as `yield` has a very specific meaning within js.

Reply:
Good call, thank you. Addressed in https://github.com/nomiclabs/hardhat/pull/1554/commits/71e5f8202f01aabe58a0aedc5bc5230f13de46e6